src/query/functions/aggregation/function_test.go

```diff
@@ -161,3 +162,43 @@ func TestAggFns(t *testing.T) {
 		}
 	}
 }
+
+var equalValuePrecisionTest = []struct {
+	name   string
+	values []float64
+}{
+	{
+		"five 1.33e-5",
+		[]float64{1.33e-5, 1.33e-5, 1.33e-5, 1.33e-5, 1.33e-5},
+	},
+	{
+		"three 13.3",
+		[]float64{13.3, 13.3, 13.3},
+	},
```

Comment:
`13.3` is from the `aggregators.test` file. I've randomly tried a handful of values, `1.33e-5` is one that makes the previous version of `varianceFn()`/`stddevFn()` fail the test.